src/Umbraco.Infrastructure/Migrations/Install/DatabaseDataCreator.cs

```diff
@@ -1606,15 +1610,15 @@ private void CreatePropertyTypeData()
                 {
                     Id = 42,
                     UniqueId = new Guid("180EEECF-1F00-409E-8234-BBA967E08B0A"),
-                    DataTypeId = Constants.DataTypes.LabelBigint,
+                    DataTypeId = Constants.DataTypes.LabelBytes,
                     ContentTypeId = 1034,
                     PropertyTypeGroupId = 52,
                     Alias = Constants.Conventions.Media.Bytes,
                     Name = "Size",
```

Comment:
This should probably read "File size" as well? It's for the Video media type
```suggestion
                    Name = "File size",
```